SK-1692-WEB-Emails-Exchange-group_management (#7122)
ovh/docs: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_groups/guide.fr-fr.md

> ## Instructions

Reviewer: ```suggestion
## En pratique
```

> ### Créez un nouveau groupe

Reviewer: ```suggestion
### Créer un nouveau groupe
```

Connectez-vous d’abord à votre [espace client OVHcloud](/links/manager). Accédez à la section `Webcloud`{.action} et sélectionnez votre service Exchange à partir de la colonne figurant sous `Microsoft`{.action} `Exchange`{.action} . Cliquez sur l'onglet `Groupes`{.action} dans le menu horizontal.

![contactgroups](images/exchange-groups-create01.png){.thumbnail .w-600 .h-600}

Cliquez sur `Créer un groupe de contacts`{.action} afin d’ouvrir une nouvelle fenêtre à partir de laquelle vous pourrez définir les paramètres du groupe :

![contactgroups](images/exchange-groups-create02.png){.thumbnail .w-600 .h-600}

- **Adresse e-mail** : Définissez une nouvelle adresse pour envoyer des messages à la liste de diffusion. Veillez à ne pas utiliser une adresse déjà fonctionnelle.

> - **Nom du Groupe** : Utilisez le nom apparaissant dans votre [espace client OVHcloud](/links/manager) et dans votre [messagerie Web d’ OVHcloud](/links/web/email) (OWA).

Reviewer: ```suggestion
- **Nom du Groupe** : Utilisez le nom apparaissant dans votre [espace client OVHcloud](/links/manager) et dans votre [messagerie Web OVHcloud](/links/web/email) (OWA).
```

- **Masquer dans Outlook** : Lorsque cette case est cochée, l’adresse du groupe ne s’affichera pas dans la liste des adresses du service Exchange.
- **Authentication requise** : Lorsque cette case est cochée, seuls les utilisateurs de la même plate-forme pourront envoyer des messages avec l’adresse du groupe.

Cliquez sur `Suivant`{.action} pour continuer.

Sur la deuxième page, sélectionnez les **Contacts** du groupe et désignez les **Administrateurs**. Ces choix se feront uniquement parmi les adresses e-mail et les contacts externes déjà répertoriés dans le service.

- **Administrateurs** : Comptes e-mail autorisés à envoyer un e-mail à l'ensemble des contacts du groupe.
- **Contacts** : Comptes e-mail qui recevront les e-mails envoyés au groupe par les administrateurs.

> [!primary]
>
> Veuillez noter que les administrateurs doivent être configurés en tant que **Contacts** afin de recevoir les e-mails du groupe.

![contactgroups](images/exchange-groups-create03.png){.thumbnail .w-600 .h-600}

Cliquez sur `Suivant`{.action} pour continuer et cliquez sur `Confirmer`{.action} pour finaliser vos choix.

> ### Gérez les groupes

Reviewer: ```suggestion
### Gérer les groupes
```

> Après la création de votre groupe, vous pouvez modifier les paramètres que vous avez défini. Pour cela, cliquez sur`...`{.action} à droite du groupe dans le tableau.

Reviewer: ```suggestion
Après la création de votre groupe, vous pouvez modifier les paramètres que vous avez définis. Pour cela, cliquez sur`...`{.action} à droite du groupe dans le tableau.
```

![contactgroups](images/exchange-groups-options01.png){.thumbnail .w-600 .h-600}

> #### Gérez les utilisateurs d'un groupe

Reviewer: ```suggestion
#### Gérer les utilisateurs d'un groupe
```

> Pour ajouter des `Contacts` à votre groupe ou définir les `Administrateurs`, cliquez sur`...`{.action} puis sur `Configurer les utilisateurs`{.action}, cochez les attribus que vous souhaitez attacher aux adresses e-mail de la colone `Compte e-mail`

Reviewer: ```suggestion
Pour ajouter des `Contacts` à votre groupe ou définir les `Administrateurs`, cliquez sur le bouton `...`{.action} puis sur `Configurer les utilisateurs`{.action}. Cochez les attributs que vous souhaitez attacher aux adresses e-mail de la colonne `Compte e-mail`.
```

> [!primary]
>
> Un groupe peut contenir un nombre maximum de 10 000 contacts.

![contactgroups](images/exchange-group-options-users01.png){.thumbnail .w-600 .h-600}

> #### Gérez les délégations d'un groupe

Reviewer: ```suggestion
#### Gérer les délégations d'un groupe
```

> De plus, l’option `Configurer les délégations`{.action} du menu s’affichera. Cette option vous permet de déléguer l’accès de la même manière qu’on le fait pour un compte Exchange. Veuillez retrouver tous les détails dans [ce guide](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation).

Reviewer: Comme il s'agit d'un nouveau H3, on ne peut pas commencer par "De plus, ..."

> De plus, l’option `Configurer les délégations`{.action} du menu s’affichera. Cette option vous permet de déléguer l’accès de la même manière qu’on le fait pour un compte Exchange. Veuillez retrouver tous les détails dans [ce guide](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation).

Reviewer: ```suggestion
De plus, l’option `Configurer les délégations`{.action} du menu s’affichera. Cette option vous permet de déléguer l’accès de la même manière qu’on le fait pour un compte Exchange. Retrouvez tous les détails dans [ce guide](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation).
```

![contactgroups](images/exchange-groups-options-delegation01.png){.thumbnail .w-600 .h-600}

> [!primary]
>

> > Veuillez noter que chaque modification à ce service peut prendre quelques minutes pour être appliquée. Vous pouvez vérifier l’état de la plupart des opérations en sélectionnant les options `Plus`{.action} et `Taches récentes`{.action} à partir du menu horizontal.

Reviewer: ```suggestion
> Veuillez noter que chaque modification de ce service peut prendre quelques minutes pour être appliquée. Vous pouvez vérifier l’état de la plupart des opérations en sélectionnant les options `Plus`{.action} et `Tâches récentes`{.action} depuis le menu horizontal.
```
